Test/lambdas/LitInt.dfy

```diff
@@ -0,0 +1,22 @@
+// RUN: %dafny /dprint:"%t.dprint" "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+const N: nat
+
+function Fibonacci(n: int): int
+{
+	if n <= 1 then 1
+	else Fibonacci(n - 2) + Fibonacci(n - 1)
```

Comment:
The indentation looks suspiciously big. You're not using tabs, right? I suggest 2 spaces for the indent.